misc/pytorch_toolkit/mammogram_screening/mammogram_screening/train_utils/loss_functions.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch import nn
+
+
+def bceLoss(mask_pred, mask, pos_weight=28):
+    eps = 1e-15
+    bce = -1.0*pos_weight*mask*torch.log(mask_pred+eps) - (1.0-mask)*torch.log(1.0-mask_pred+eps)
+
+    return torch.mean(bce)
+
+criterion = nn.CrossEntropyLoss()
+
+def ceLoss(pred, target):
+    return criterion(pred, target)
```

Comment:
```suggestion

def ceLoss(pred, target):
    criterion = nn.CrossEntropyLoss()
    return criterion(pred, target)
```
since this variable is relevant only to a certain function, it should be moved into it